cmd/bucket-handlers.go

```diff
@@ -182,6 +182,47 @@ func initFederatorBackend(buckets []BucketInfo, objLayer ObjectLayer) {
 	wg.Wait()
 }
 
+// GetBucketPanFSPathHandler - GET Bucket path in PanFS filesystem.
+// -------------------------
+// This operation returns path to the bucket in scope of PanFS filesystem.
+func (api objectAPIHandlers) GetBucketPanFSPathHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := newContext(r, w, "GetBucketPanFSPath")
```

Comment:
I'm not sure that I understand correctly what you mean. This function just fetches the `panfsPath` from the bucket metadata and returns it. So it doesn't matter which value it will return either valid path or empty string (actually such situation shouldn't happen)

Reply:
I think it should work fine. If I'm not mistaken this function actually never returns `"", err"`